hubconf.py

```diff
@@ -1,14 +1,24 @@
 # Optional list of dependencies required by the package
 dependencies = ['torch']
 
+# classification
 from torchvision.models.alexnet import alexnet
 from torchvision.models.densenet import densenet121, densenet169, densenet201, densenet161
 from torchvision.models.inception import inception_v3
 from torchvision.models.resnet import resnet18, resnet34, resnet50, resnet101, resnet152,\
     resnext50_32x4d, resnext101_32x8d, wide_resnet50_2, wide_resnet101_2
 from torchvision.models.squeezenet import squeezenet1_0, squeezenet1_1
 from torchvision.models.vgg import vgg11, vgg13, vgg16, vgg19, vgg11_bn, vgg13_bn, vgg16_bn, vgg19_bn
-from torchvision.models.segmentation import fcn_resnet101, deeplabv3_resnet101
 from torchvision.models.googlenet import googlenet
 from torchvision.models.shufflenetv2 import shufflenet_v2_x0_5, shufflenet_v2_x1_0
 from torchvision.models.mobilenet import mobilenet_v2
+from torchvision.models.mnasnet import mnasnet0_5, mnasnet0_75, mnasnet1_0, \
+    mnasnet1_3
+
+# segmentation
+from torchvision.models.segmentation import fcn_resnet50, fcn_resnet101, \
+    deeplabv3_resnet50, deeplabv3_resnet101
+
+# object detection
+from torchvision.models.detection import fasterrcnn_resnet50_fpn, \
+    maskrcnn_resnet50_fpn, keypointrcnn_resnet50_fpn
```

Comment:
I think detection models won't work as expected with torchhub.
This is because they require compiled C++ extensions, which do not get compiled in Hub, and would instead try to pick the ops from torchvision.

Reply:
Will remove detection models. Segmentation models are okay?

>which do not get compiled in Hub, and would instead try to pick the ops from torchvision.

Ah why they worked on my local env.

Reply:
because you already have torchvision installed on your system. But if the version of torchvision that you had installed in your system was quite old (say 0.3.0), then the model wouldn't work because the interface for the custom ops has since changed.